src/paratii.js

```diff
@@ -80,6 +85,87 @@ class Paratii extends ParatiiCore {
   getAccount () {
     return this.eth.getAccount()
   }
+  /**
+   * Pings the provider to which the web3 is configured to connect to (see the set up in paratii.eth.js constructor)
+   * @return {Promise} that resolves in a boolean
+   */
+  async pingEth () {
+    return new Promise(resolve => {
+      this.eth.web3.eth.net.isListening()
+        .then(() => {
+          resolve(true)
+        })
+        .catch(e => {
+          resolve(false)
+        })
+    })
+  }
+  /**
+   * Checks if the IPFS local node is running
+   * @return {Promise} that resolves in a boolean
+   */
+  async checkIPFSState () {
+    return new Promise(resolve => {
+      this.ipfs.getIPFSInstance()
+        .then((ipfsInstance) => {
+          if (ipfsInstance.state.state() === 'running') {
+            resolve(true)
+          } else {
+            resolve(false)
+          }
+        })
+        .catch(e => {
+          resolve(false)
+        })
+    })
+  }
+  /**
+   * Requests a link to see if it's up (Easily adds a dozen seconds to check the status)
+   * @param {string} linkToCheck
+   * @return {Promise} that resolves in a boolean
+   */
+  async requestStatusCode (linkToCheck) {
```

Comment:
this is very confusingly named - it is checking if the status code is 200, not return the status code from the request (or equesting the status code).

Reply:
Done in: https://github.com/Paratii-Video/paratii-js/pull/266/commits/114c3ada630021eb9c485f2b2fc010df317aa966